pkg/backends/jmeter/config.go

```diff
@@ -0,0 +1,13 @@
+package jmeter
+
+// Config specific to JMeter backend
+type Config struct {
+	MasterCPULimits      string `envconfig:"JMETER_MASTER_CPU_LIMITS" default:"2000m"`
```

Comment:
Should it be another way around?
So we have small defaults and we can override those values for production load testing.

Reply:
I think the current default numbers are fair; we only adjust the numbers in integration test because github actions (free) runner is really limited in resource.  
In production you do want somewhat high resource requests and limits, otherwise your load generator might "fail" before the service does, and you get innacurate results.

Reply:
I don't agree with that tbh. Can we even leave that empty here and override only if it is needed?
Because for example I want to run Kangal locally but I can't because it requires high specs by default. But for my small test my local laptop specs should be fine - so not really needed to set any limits at all.

Reply:
Resource limits and requests are no longer required as of 1fb2cab.